tslib/src/functions/util-submit-check/index.ts

```diff
@@ -1,30 +1,49 @@
 
 import { AzureFunction, Context, HttpRequest } from '@azure/functions'
-import * as lz from 'lz-string'
-import submittedCheck from '../../schemas/submitted-check.v3'
-import completeCheckPayload from '../../schemas/complete-check-payload'
 import config from '../../config'
+import { FakeSubmittedCheckMessageGeneratorService } from './fake-submitted-check-generator.service'
+import { SchoolChecksDataService } from './school-checks.data.service'
 
-const httpTrigger: AzureFunction = function (context: Context, req: HttpRequest): void {
+const fakeSubmittedCheckBuilder = new FakeSubmittedCheckMessageGeneratorService()
+const liveSchoolChecksDataService = new SchoolChecksDataService()
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
   if (!config.DevTestUtils.TestSupportApi) {
-    context.log('exiting as not enabled (default behaviour)')
+    context.log('exiting as config.DevTestUtils.TestSupportApi is not enabled (default behaviour)')

```

Comment:
I would strongly suggest that all functions logs are prefixed by the name of the function.